src/data/real/ereal.lean

```diff
@@ -165,6 +165,39 @@ begin
   { simpa using hx },
 end
 
+lemma ereal.to_real' {x : ereal} (h : x ≠ ⊥) (h' : x ≠ ⊤) :
+  ∃ (c : ℝ), (c : ereal) = x := by {rcases x with rfl|rfl|x;tauto}
```

Comment:
This lemma doesn't look useful to me - it's just a weaker version of `coe_to_real` above, which says 'there is a `c` and it is called `x.to_real`

Reply:
I was finding it useful in doing ```obtain  <c, hc> := ereal.to_real' ...```. But it's very likely that you are right and they can be avoided.